cluster/log-dump.sh

```diff
@@ -152,10 +159,19 @@ function kubectl-guess-nodes() {
   kubectl get node -ojsonpath --template="${ips_and_images}" | grep -v kube-apiserver | cut -f1 -d" "
 }
 
-function dump_master() {
-  local master_name
+function awscli-instances() {
+  local -r role=$1
+  for region in "${awscli_regions[@]}"; do
+    aws ec2 describe-instances --region "${region}" --filter "Name=tag-value,Values=$(kubectl config current-context)" "Name=instance-state-name,Values=running" "Name=tag-key,Values=k8s.io/role/${role}" | jq -r '.["Reservations"][]["Instances"][]["PublicDnsName"]'
```

Comment:
The AWS CLI query support is really good, FYI.  I think the equivalent would be `--query Reservations[].Instances[].PublicDnsName`.  Maybe `--output text` as well depending on how you want it formatted.

Strictly I think you probably also want to do `Name=tag:KubernetesCluster,Values=$(kubectl config current-context)`, but I'm not sure if there's a second tag you're trying to match on also.